strings/tips/xeno.txt

```diff
@@ -1,59 +1,56 @@
-As a xeno, you can freely melt barricades while in boiler or defiler gas as long as you can click on the right spot, the gas is also a really good hiding spot to launch ambushes from without marines being aware of you.
+As a xeno, you can freely melt barricades while in boiler or defiler gas as long as you can click on the right spot. The gas is also a really good hiding spot to launch ambushes from without marines being aware of you.
 As a xeno, always thank your drones and hivelords for supporting the hive!
-As a xeno, you can drag any items, but you must wait before you drag the desirable item!
+As a xeno, you can drag any items, but you must wait through an actionbar before you can do so!
 As a xeno, do not fight as a young xenomorph.
 Most alien deaths are caused by over-aggression. Rein yourself in, or you may find yourself dying round after round.
 As a xeno, if you have difficulty clicking marines, try using bump slash. You can use movement keys to slash marines. That said, clicking marines to slash them is faster than bump slashing!
-It is EXTREMELY IMPORTANT that you as a xenomorph have your abilities set up on keybinds. Ask for help if you don't know how in XOOC or mhelp.
-In the xeno tab, you can use the crawl through vent verb to quickly find and use vents in the floor without having to alt-click them. You can also keybind it!
+It is EXTREMELY IMPORTANT that you as a xenomorph have your abilities set up on keybinds. Ask for help if you don't know how in XOOC or mentorhelp.
+In the xeno tab, you can use the "crawl through vent" verb to quickly find and use vents in the floor without having to alt-click them. You can also keybind it!
 As a xeno, it is faster to re-fill an empty acid well than rolling on the ground to put out a fire.
-As a xeno, acid well extinguishes fire and explode into an acid cloud when destroyed. The queen, the shrike, praetorians, boilers, and hivelords can make acid wells.
+As a xeno, acid wells extinguish fire and explode into an acid cloud when destroyed. The queen, the shrike, praetorians, boilers, and hivelords can make acid wells.
 The queen, shrike, and hivelord have access to Queen's Blessing, a menu for the hive to use psychic points to give xenomorphs a cutting edge in the battlefield.
 If you have acid spit, you can spit at a welding fuel tank to blow it up. If the welding fuel is near hostiles, you might give them a pleasant surprise!
 As a xeno, check hive status often.
 As a xeno, you've got pheromones that can help the hive.
-As a xeno, you can pounce past window frames and barricades without barbed wires.
-As a xeno, pouncing into a barricade with barbed wire stuns you.
+As a runner or hunter, you can pounce past window frames and barricades without barbed wires.
```

Comment:
Many xenos can leap over barricades. Defender, warrior, prae, ravager off the top of my head.